Adding some CBIO setup (#14)
ilifu/ilifu_user_docs: docs/cbio/setup.md

# CBIO setup

* User homes are in `/users/`. Not sure if this is being backed up, assume not.

> * CBIO users will be using the CephFS parallel file partition. The BeeGFS partitions are for the astronomer users. Our space is in `/ceph/cbio/`. Directories that have been setup

Reviewer: ```suggestion
* CBIO users will be using the CephFS parallel file partition. The BeeGFS partitions are for the astronomer users. Our space is in `/data/projects/cbio/`. Directories that have been setup
```

>   * `/ceph/cbio/users` - a user's private scratch space

Reviewer: ```suggestion
  * `/data/projects/cbio/users` - a user's private scratch space
```

>   * `/ceph/cbio/projects` - project spaces will be created in here

Reviewer: ```suggestion
  * `/data/projects/cbio/projects` - project spaces will be created in here
```

>   * `/ceph/cbio/dbs` - annotation and reference databases will go in here

Reviewer: ```suggestion
  * `/data/projects/cbio/dbs` - annotation and reference databases will go in here
```

>   * `/ceph/cbio/datasets` - mostly access controlled datasets will go in here e.g. SAHGP, Baylor, AGVP.

Reviewer: ```suggestion
  * `/data/projects/cbio/datasets` - mostly access controlled datasets will go in here e.g. SAHGP, Baylor, AGVP.
```

>   * `/ceph/cbio/soft` - some software that are needed to run workflows will be installed here, most over our other software will be encapsulated into images.

Reviewer: ```suggestion
  * `/ceph/cbio/soft` - some software that are needed to run workflows will be installed here, most of our other software will be encapsulated into singularity images.
```

>   * `/ceph/cbio/images` - singularity images that need to be shared would be stored here.

Reviewer: ```suggestion
  * `/data/projects/cbio/images` - singularity images that need to be shared would be stored here.
```

> Some things to note

Reviewer: ```suggestion
Some things to note:
```

> * CBIO are allowed to use 400TB on the CephFS partition. There is no user, group or project restrictions in place currently.

Reviewer: ```suggestion
* CBIO is allowed to use 400TB on the CephFS partition. There are no user, group or project restrictions in place currently.
```

> * Compute resource restrictions are not in place. There are plans to use the SLURM fair ussage to share resources according to initial funding been provided by astronomy, CBIO and DST respectively.

Reviewer: ```suggestion
* Compute resource restrictions are not in place. There are plans to use [SLURM's fair-share factor](https://slurm.schedmd.com/priority_multifactor.html#fairshare)  to share resources according to initial funding been provided by astronomy, CBIO and DST respectively.
```

* Currently there are only 1216 cores available on the standard SLURM cluster. This will probably be increased after the pilot phase.
* CBIO can access an additional 320 cores on the OpenStack environment.

> Examples

Reviewer: ```suggestion
## Examples
```

* [Running a Nextflow pipeline on the Ilifu standard SLURM setup](https://github.com/grbot/run-fastqc/tree/ilifu).

> [Future work and data requirements](http://web.cbio.uct.ac.za/~gerrit/slides/CBIO-Ilifu-compute-storage-and-transfer-setup.pdf)

Reviewer: ```suggestion
## Future work and data requirements
[CBIO/Ilifu compute, storage and transfer setup requirements](http://web.cbio.uct.ac.za/~gerrit/slides/CBIO-Ilifu-compute-storage-and-transfer-setup.pdf)
```
